test/lib/barnes/puma_stats_test.rb

```diff
@@ -27,12 +27,32 @@ def test_cluster_no_values
   end
 
   def test_missing_key_single
-    stat = stat_value({ "backlog" => 0, "running" => 0, "pool_capacity" => 16 }, "does_not_exist")
+    stats_hash = { "backlog" => 0, "running" => 0, "pool_capacity" => 16 }
```

Comment:
I guess I wouldn't expect `backlog` to exist here?

Reply:
The `backlog` key is still being sent via puma, we're just not doing anything with it.